editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java

```diff
@@ -241,9 +243,20 @@ private void installDefaultFacets(IProject project, MavenProject mavenProject, I
 	        installCamelFacet(fproj, fpwc, camelVersion, monitor);
 	        fpwc.commitChanges(monitor);
 	        configureNature(project, mavenProject, monitor); 
+	        updateMavenProject(project, monitor);
 	    }
 	}
 	
+	 private void updateMavenProject(IProject project, IProgressMonitor monitor) throws CoreException {
+        // MANIFEST.MF is probably not built yet
+        if (project != null) {
+            // update the maven project so we start in a deployable state
+            // with a valid MANIFEST.MF built as part of the build process.
+            Job updateJob = new UpdateMavenProjectJob(new IProject[] {project });
+            updateJob.schedule();
```

Comment:
to be honest I don't like very much the idea of calling updateMavenProjectJob.
I think that either the m2e connector has a bug or the "InterruptedException" (see https://issues.jboss.org/browse/FUSETOOLS-1899) stack that we see since the beginning is blocking some Maven configurator to work correctly